Content.Server/_NF/Medical/MedicalBountySystem.cs

```diff
@@ -185,8 +188,9 @@ private void RedeemMedicalBounty(EntityUid uid, MedicalBountyRedemptionComponent
         }
         else if (bountyPayout > 0)
         {
-            // Use SpawnMultiple in case spesos ever have a limit.
-            _stack.SpawnMultiple("SpaceCash", bountyPayout, Transform(uid).Coordinates);
+            var stackUid = _stack.Spawn(bountyPayout, "SpaceCash", Transform(uid).Coordinates);
```

Comment:
```suggestion
            var stackUid = _stack.Spawn(bountyPayout, "Credit", Transform(uid).Coordinates);
```
The SpaceCash is the entity name, but the stack prototype name is Credit.

Reply:
Changes applied.  Thank you.